lib/membrane_aac_format/remote.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Membrane.RemoteStream.AAC do
+  @moduledoc """
+  Capabilities defining Remote AAC Stream with known audio decoder configuration
```

Comment:
The struct in this module is not "capabilities". I know it has been confusing, but at some point (if you're interested, the discussion is here: https://github.com/membraneframework/membrane_core/issues/236) we decided to define this properly:
1. Caps = capabilities = a set of formats supported by a pad, defined when defining a pad
2. Format = a specific media format that is described by a struct containing its properties

tl;dr This is the format definition